### PR TITLE
Fix For btcDestinationAddress from Hash160 to Base58 Format

### PR DESCRIPTION
**Pull request by kelvinator07**

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const btcAddressFromPublicKeyHash = (pubKeyHash, network) => {
+    return bitcoin.payments.p2pkh({
+        hash: Buffer.from(stripHexPrefix(pubKeyHash), "hex"),
+        network: getNetwork(network),
+    }).address;
+};
+
+const getNetwork = (network) => {
+    return network === "mainnet" ? bitcoin.networks.MAINNET : bitcoin.networks.testnet;
```
Comment (marcos-iov) [inline utils.js:11]: Did you test the tool against mainnet? I believe the name for mainnet network is `bitcoin`. See https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/src/networks.d.ts

**Review by marcos-iov — COMMENTED** [review]
We should add a test to ensure the btc address hash is correctly decoded. To do this I would add a `release_request_received` event to one of the mocked transactions. Then we can validate the address decoded corresponds to the hash in the event

File: utils.js (hunk 1 above)
Comment (kelvinator07) [reply utils.js:11]: Yes I Did Test It With Mainnet

Comment (kelvinator07) [reply utils.js:11]: For Some Weird Reason, It Works Well With bitcoin.networks.MAINNET or bitcoin.networks.mainnet or bitcoin.networks.bitcoin, The Latter Is The Correct One

**Comment by kelvinator07** [comment]
> We should add a test to ensure the btc address hash is correctly decoded. To do this I would add a `release_request_received` event to one of the mocked transactions. Then we can validate the address decoded corresponds to the hash in the event

Unit Test Updated With `release_request_received` event

File: utils.js (hunk 1 above)
Comment (marcos-iov) [reply utils.js]: It probably defaults to mainnet

**Review by josedahlquist — APPROVED** [review]
LGTM, just a minor comment, if this library is already in use we should upgrade its version.

Please tag the release once you merge it

**Comment by kelvinator07** [comment]
> LGTM, just a minor comment, if this library is already in use we should upgrade its version.
> 
> Please tag the release once you merge it

Updated Version To 0.1.1-beta

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**